.trampolinerc

```diff
@@ -47,7 +47,7 @@ if [[ -z "${TRAMPOLINE_IMAGE:-}" ]]; then
 fi
 
 if [[ -z "${TRAMPOLINE_DOCKERFILE:-}" ]]; then
-    TRAMPOLINE_DOCKERFILE=".kokoro/docker/Dockerfile"
+    TRAMPOLINE_DOCKERFILE=""
```

Comment:
I'd love to have a comment explaining what setting this variable does, so we don't have to think too much about this in the future, just have the info on a plate.

Reply:
This tells which dockerfile you can specify to use for the Kokoro job, iirc. We can add a comment in a followup PR.